draft/integrating-events-operator/integrating-events-operator.md

```diff
@@ -0,0 +1,347 @@
+---
+permalink: /guides/integrating-events-operator/
+layout: guide-markdown
+title: Integrating the events operator
+duration: 30 minutes
+releasedate: 2020-04-26
+description: Explore how to manage the flow of GitHub events to multiple pipelines
+tags: ['Pipelines', 'Stack']
+guide-category: pipelines
+---
+
+<!-- Note:
+> This repository contains the guide documentation source. To view
+> the guide in published form, view it on the [website](https://kabanero.io/guides/{projectid}.html).
+-->
+
+<!--
+//
+//	Copyright 2020 IBM Corporation and others.
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+//
+-->
+
+## What you'll learn
+
+In this guide, you will learn how to integrate the events operator to manage the flow of GitHub organization events through multiple pipelines. By setting up the webhook mediator, you will manage event flow based on the semantic versioning of your application stack projects.
+
```

Comment:
The main use cases are:
1) Support for Github organizational webhook, which only needs to be created once, versus having to create webhooks for every repository within the organization.
2) Support calling the version of Tekton pipeline that best matches the Appsody stack in the github repository, using semantic versioning.